test/test_tae/test_exec_tae_run.py

```diff
@@ -81,6 +83,85 @@ def test_start_crash_first_run(self, test_run):
         self.assertRaises(
             FirstRunCrashedException, eta.start, config={}, instance=1)
 
+    @mock.patch.object(ExecuteTARun, 'run')
+    def test_start_tae_return_nan_inf(self, test_run):
+        '''
+            test nan-handling and inf-handling
+        '''
+        def get_tae(obj):
+            """ Create ExecuteTARun-object for testing. """
+            scen = Scenario(scenario={'cs': ConfigurationSpace(), 'run_obj': obj,
+                'abort_on_first_run_crash': '0', 'cutoff_time': '10'}, cmd_args=None)
+            stats = Stats(scen)
+            stats.start_timing()
+            eta = ExecuteTARun(ta=lambda *args: None, stats=stats, run_obj=obj)
+            # Add first run to not trigger FirstRunCrashedException
+            test_run.return_value = StatusType.SUCCESS, 1, 1, {}
+            eta.start(config={}, instance=1)
```

Comment:
What is this call doing here? I assume it prevents an exception if the first run crashes. If yes, `stats.ta_run += 1` would me more intuitive.